src/Components/UI/Navigation/LeftNavBar.jsx

```diff
@@ -1,37 +1,50 @@
-import { Link } from "react-router-dom";
+import { Link, useLocation } from "react-router-dom";
 import "./LeftNavBar.css";
+import ideaIcon from "../../../Data/Images/Icons/idea.svg";
+import labelIcon from "../../../Data/Images/Icons/label.svg";
+import archiveIcon from "../../../Data/Images/Icons/archive.svg";
+import trashIcon from "../../../Data/Images/Icons/trash.svg";
+import userIcon from "../../../Data/Images/Icons/user.svg";
```

Comment:
Suggestion: You could create a common icons file to import all icons in just one line, it will make your code look much cleaner.

Reply:
Noted, will update it.